distributed/worker.py

```diff
@@ -2680,6 +2681,13 @@ def get_call_stack(self, comm=None, keys=None):
         result = {k: profile.call_stack(frame) for k, frame in frames.items()}
         return result
 
+    def _notify_transition(self, key, start, finish, **kwargs):
+        for plugin in self.plugins.values():
+            try:
+                plugin.transition(key, start, finish, **kwargs)
```

Comment:
Old plugins in existing user code may not have this method set.  Let's allow this for now.

```suggestion
                plugin.transition(key, start, finish, **kwargs)
            except AttributeError:
                pass
```